### ISSUE TITLE
Duplicate TV Show entries in Home

### ISSUE THREAD

**Issue by hamburglar2160** [jellyfin/jellyfin#2698]
i've got a TV show in a single media library but in two different locations. Five seasons are on one HDD and another season on another. On the Home screen under the "Latest TV Shows" section the server will report **two duplicate entries**. The TV shows detail page does display all of the episodes as intended.

**Comment by progtologist** [comment]
The folder structure for your library is specified in [this](https://jellyfin.org/docs/general/server/media/shows.html) webpage. If you have the same TV Show in two different folders, then you must combine them under the same folder.

**Comment by hamburglar2160** [comment]
Yet the server handles them in separate folders just fine. Certainly it could be changed to report the same show only once under Latest.

**Comment by oddstr13** [comment]
Duplicate of #904
